cmd/podman/system/connection/add.go

```diff
@@ -236,77 +279,37 @@ func getUDS(cmd *cobra.Command, uri *url.URL) (string, error) {
 			}
 		}
 	}
-
 	var authMethods []ssh.AuthMethod
 	if len(signers) > 0 {
 		var dedup = make(map[string]ssh.Signer)
-		// Dedup signers based on fingerprint, ssh-agent keys override CONTAINER_SSHKEY
 		for _, s := range signers {
 			fp := ssh.FingerprintSHA256(s.PublicKey())
 			if _, found := dedup[fp]; found {
-				logrus.Debugf("Dedup SSH Key %s %s", ssh.FingerprintSHA256(s.PublicKey()), s.PublicKey().Type())
+				logrus.Debugf("Dedup SSH Key %s %s", ssh.FingerprintSHA256(s.PublicKey()), s.PublicKey().Type()) // public key info and validation
```

Comment:
I don’t understand what this comment means; is it saying something non-obvious about the contents of the debug log? Something entirely different?